app/assets/v2/js/cart-ethereum-polygon.js

```diff
@@ -138,6 +134,12 @@ Vue.component('grantsCartEthereumPolygon', {
       this.polygon.checkoutStatus = 'depositing';
     },
 
+    getBulkCheckoutAddress() {
+      return appCart.$refs.cart.network === 'mainnet'
+        ? '0xb99080b9407436eBb2b8Fe56D45fFA47E9bb8877'
+        : '0x3E2849E2A489C8fE47F52847c42aF2E8A82B9973';
```

Comment:
Is `0x3E2849E2A489C8fE47F52847c42aF2E8A82B9973` supposed to be the BulkCheckout contract on Polygon? It seems like there's no contract there: https://polygonscan.com/address/0x3E2849E2A489C8fE47F52847c42aF2E8A82B9973

Reply:
yea... that's for mumbai.

[0xb99080b9407436eBb2b8Fe56D45fFA47E9bb8877](https://polygonscan.com/address/0xb99080b9407436eBb2b8Fe56D45fFA47E9bb8877) is for mainnet

Reply:
Ah right these are only the polygon addresses, makes sense! I was thinking the first address was L1, sorry

And just to confirm `mainnet` is what the network name will be when on mainnet polygon? I would guess it's `polygon` since `mainnet` would be misleading/confusing, but I'm not sure so figured I'd ask

Reply:
yes polygon also uses `mainnet` as network name and `testnet` for mumbai network.

surpising that was pretty straightforward with our system... `testnet` was more complicated to stick with

Reply:
Heh interesting, well good to know, thanks!